### PR TITLE
Expand the Lambda tutorial with inserting function

### PR DESCRIPTION
**Pull request by zseta**
Expand the Lambda tutorial with inserting function using AWS Lambda + API Gateway to insert data into TimescaleDB

https://docs-dev.timescale.com/docs-lambda-post-api/timescaledb/lambda-post-api/tutorials/aws-lambda/create-data-api/#create-a-new-lambda-function-to-insert-data-into-the-database

### CODE REVIEW

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 1]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
 [custom-lambda-integration]: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-custom-integrations.html
+
+## Create a Lambda function to insert data into the database
+Now that you know how to create a `GET` API for your database, let's see how
+to create a `POST` API which allows you to insert data into the database using 
+a JSON payload.
+
+<procedure>
+
+### Create a Lambda function to insert data into the database
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:347]:
```suggestion
### Creating a Lambda function to insert data into the database
```

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 2]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
 [lambda-layers]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html
 [lambda-psycopg2]: https://github.com/jkehler/awslambda-psycopg2
 [custom-lambda-integration]: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-custom-integrations.html
+
+## Create a Lambda function to insert data into the database
+Now that you know how to create a `GET` API for your database, let's see how
+to create a `POST` API which allows you to insert data into the database using 
+a JSON payload.
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:343]:
```suggestion
When you have created the `GET` API for your database, you can
create a `POST` API. This allows you to insert data into the database
with a JSON payload.
```

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 3]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
+        }
+    
+      ```
+1. Upload the function to AWS Lambda:
+  ```bash
+  zip insert_function.zip insert_function.py
+  aws lambda create-function --function-name insert_function \
+  --runtime python3.8 --handler function.lambda_handler \
+  --role <ARN_LAMBDA_ROLE> --zip-file fileb://insert_function.zip
+  ```
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:395]: Indent

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 4]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
+1. Create a new API Gateway, called `InsertApi`:
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:396]:
```suggestion
1. Create a new API gateway called `InsertApi`:
```

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 5]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
+    ```bash
+    aws apigateway create-rest-api --name 'InsertApi' --region us-east-1
+    ```
+1.  Retrieve the `id` of the root resource, to add a new POST endpoint:
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:400]:
```suggestion
1.  Retrieve the `id` of the root resource, and add a new POST endpoint:
```

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 6]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
+    aws apigateway create-resource --rest-api-id <API_ID> --region us-east-1 
+    --parent-id <RESOURCE_ID> --path-part insert
+    {
+        "id": "arabc2",
+        "parentId": "r5fc0ufn0h",
+        "pathPart": "insert",
+        "path": "/insert"
+    }
+    ```
+1.  Create a POST request for the /insert resource:
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:432]:
```suggestion
1.  Create a POST request for the `insert` resource:
```

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 7]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
+```bash
+curl -X POST -H "Content-Type: application/json" -d @./post.json 
+https://h45kwepq8g.execute-api.us-east-1.amazonaws.com/test_post_api/insert_function
+```
+
+If everything is working properly, the content of your JSON payload file gets 
+inserted into the database.
+
+|time|symbol|price_open|price_high|price_low|price_close|trading_volume|
+|---|---|---|---|---|---|---|
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:512]:
```suggestion
|-|-|-|-|-|-|-|
```

File: timescaledb/tutorials/aws-lambda/create-data-api.md [hunk 8]
```diff
@@ -336,3 +336,184 @@ this example, it's the latest stock price of MSFT (Microsoft) in JSON format.
...
+|2021-11-12 21:00:00|AAPL|149.8|149.73|149.73|149.81|17291|
+|2021-11-12 21:00:00|MSFT|337.15|337.15|337.15|337.15|562|
+|2021-11-12 21:00:00|FB|341.35|341.35|341.3|341.3|556|
+
+
+
+
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/create-data-api.md:519]:
```suggestion
```

**Review by Loquacity — CHANGES_REQUESTED** [review]
Just a couple of small things. Nice work!

File: timescaledb/tutorials/aws-lambda/create-data-api.md (hunk 3 above)
Comment (zseta) [reply timescaledb/tutorials/aws-lambda/create-data-api.md]: https://github.com/timescale/docs/pull/631/commits/e3b2e6afaba56c2bb2f4557d5317d31cab7bed0d

**Review by Loquacity — APPROVED** [review]

**Pull request merged**